tslint.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
I would prefer `Eslint` since `TSLint` support is ending soon.

Reply:
Updated.